contrib/redwood/plugins.yml

```diff
@@ -24,3 +24,11 @@
   description: |
     Tutor plugin to install Meilisearch and enable course content search in Studio.
     See the README on GitHub for further setup instructions.
+
+- name: k8s_harmony
```

Comment:
Please insert this plugin in alphabetical order. (yes, I know, this wasn't required before... sorry it wasn't clear)